File: app/src/main/java/com/example/mechrevo/xysdemoset/activity/MainActivity.java
package com.example.mechrevo.xysdemoset.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.example.mechrevo.xysdemoset.R;
import com.example.mechrevo.xysdemoset.view.StatusChangeView;


public class MainActivity extends AppCompatActivity implements View.OnClickListener {

    StatusChangeView statusChangeView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        statusChangeView = findViewById(R.id.statusChangeView);
        statusChangeView.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        statusChangeView.onHandleText();
    }
}

File: okhttpentra/src/main/java/com/example/okhttpentra/MainActivity.kt
package com.example.okhttpentra

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.example.okhttpentra.net.Response
import java.net.InetSocketAddress
import java.net.Socket
import javax.net.ssl.SSLSocketFactory

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val socket = SSLSocketFactory.getDefault().createSocket()
        //链接
        socket.connect(InetSocketAddress("",1))

        val outputStream = socket.getOutputStream()
        val inputStream = socket.getInputStream()

        val socketOutPutStream = socket.getOutputStream()

    }

}
